Makefile

```diff
@@ -68,18 +70,29 @@ $(build_log): cli-linux cli-darwin cli-windows
 	@echo Build success at: `/bin/date "+%Y-%m-%d---%H-%M-%S"` >> $(build_log);
 
 cli-linux: $(merge_log)
-	GO111MODULE=on go run $(gox) -osarch=linux/amd64 -output=$(bin_linux) ./$(merge_src_dir)
+	GO111MODULE=on CGO_ENABLED=0 go run $(gox) -osarch=linux/amd64 -output=$(bin_linux) ./$(merge_src_dir)
```

Comment:
Since golang v1.16, the default behaviour is to use go modules (which is what `GO111MODULE=on` does) - we're using 1.17 so we can remove that from the commands if we like.

Additionally latest go version is 1.19, so we could bump to that version if it doesn't cause any issues. (I forgot to do this with latest SR release, I should've done so!)

Neither are terribly important IMO, just suggestions.

Reply:
both are good suggestions, thanks, I'll bump golang version and remove unnecessary `GO111MODULE`s